spec/system/simple_query_spec.rb

```diff
@@ -12,6 +12,7 @@
     it "returns a response with the informaton for the address" do
       entries = subject.entries
       expect(entries[0].name).to eq("Farnell AG")
+      expect(entries[0].last_name).to eq("Farnell AG")
```

Comment:
@Liberatys Do we want `last_name` only to be set for `type == Person`? I'd personally slightly prefer that, but I'm totally fine with setting it to the company name, if that is your intention!

Reply:
sounds like a good idea 👍🏻 will limit it to type == Person

Reply:
I figured if we set last_name for a person we can also set company for an organisation :D